src/unstract/sdk/audit.py

```diff
@@ -84,6 +85,7 @@ def push_usage_data(
             response = requests.post(
                 url, headers=headers, json=data, timeout=30
             )
+            token_counter.reset_counts()
```

Comment:
NIT: I'm assuming this resets the token usage for the counter - it might be good to place this in a `finally` block to execute irrespective of whether an error occurred or not

Reply:
Done